### ISSUE TITLE
Use string format keyword args instead of .replace

### ISSUE THREAD

**Issue by wrhansen** [oskarcieslik/Jeff#2]
Nice little tool, but I had a question about the implementation. Why not just use string's keyword `format` replacement fields in the licenses instead of the .replace() for loop you have? For example it looks like you've got four special args that you replace in the licences: _project_, _email_, _name_, and _year_. Rather than create your own replacement system using `[project]` in a license file you can use `{project}`. Then your `_fill_license` function simply returns the formatted template like:

``` python
credentials = {
    'project': arguments['--project'] or '',
    'email': arguments['--email'] or '',
    'name': arguments['--name'] or getpass.getuser(),
    'year': arguments['--year'] or str(date.today().year)
}
return license_template.format(**credentials)
```

**Comment by rpunkfu** [comment]
Hi @wrhansen, thanks for contributing to Jeff! :) This is an awesome idea, I'm starting to review your PR now :)

**Comment by rpunkfu** [comment]
Fixed by #3.
